### DISCUSSION TITLE
What does --prod do?

### DISCUSSION THREAD

**Discussion by coatsbj** [aws/copilot-cli#2158] [Q&A]
The --prod flag for environment initialization is mentioned, but i have yet to be able to find anything, anywhere, indicating what it actually _does_.

I'd love it if --prod would, rather than deploying a service to `[servicename].[env].[app].[custom-domain]`, would instead deploy it to `[servicename].[app].[customdomain]`...but I have a sneaking suspicion that's not the point.  :-)

Anyone know anything about it?

**Comment by iamhopaul123** [comment]
Hello @coatsbj. `--prod` currently is just a placeholder and isn't in use. Sorry about that.

**Reply by kohidave** [reply]
Well, we have some plans for it in the future. Something like psudo-selectors in the environment overrides (all prod environments use `this` configuration), enabling termination protection for prod resources by default, etc. 

We just haven't implemented those things yet :)
